### ISSUE TITLE
crash on mesage send from bubble

### ISSUE THREAD

**Issue by theotheroracle** [deku-messaging/Deku-SMS-Android#130]
```
FATAL EXCEPTION: main
Process: com.afkanerd.deku, PID: 13050
java.lang.IllegalArgumentException: Tmp detached view should be removed from RecyclerView before it can be recycled: MessageSentViewHandler{72c4d21 position=0 id=-1, oldPos=-1, pLpos:-1 update tmpDetached no parent} androidx.recyclerview.widget.RecyclerView{69d2e5c VFED..... ......ID 0,0-990,703 #7f090252 app:id/single_messages_thread_recycler_view}, adapter:com.example.swob_deku.Models.Messages.SingleMessagesThreadRecyclerAdapter@c34f565, layout:androidx.recyclerview.widget.LinearLayoutManager@66f723a, context:android.view.ContextThemeWrapper@4247351
	at androidx.recyclerview.widget.RecyclerView$Recycler.recycleViewHolderInternal(RecyclerView.java:6620)
	at androidx.recyclerview.widget.RecyclerView.removeAnimatingView(RecyclerView.java:1525)
	at androidx.recyclerview.widget.RecyclerView$ItemAnimatorRestoreListener.onAnimationFinished(RecyclerView.java:13109)
	at androidx.recyclerview.widget.RecyclerView$ItemAnimator.dispatchAnimationFinished(RecyclerView.java:13611)
	at androidx.recyclerview.widget.SimpleItemAnimator.dispatchAddFinished(SimpleItemAnimator.java:302)
	at androidx.recyclerview.widget.DefaultItemAnimator$5.onAnimationEnd(DefaultItemAnimator.java:247)
	at android.view.ViewPropertyAnimator$AnimatorEventListener.onAnimationEnd(ViewPropertyAnimator.java:1115)
	at android.animation.Animator$AnimatorListener.onAnimationEnd(Animator.java:600)
	at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1333)
	at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1575)
	at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:328)
	at android.animation.AnimationHandler.-$$Nest$mdoAnimationFrame(Unknown Source:0)
	at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:86)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1229)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
	at android.view.Choreographer.doCallbacks(Choreographer.java:899)
	at android.view.Choreographer.doFrame(Choreographer.java:827)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7924)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
``
android 13 : 20-20231004-microG-enchilada
version : 0.25.0 F-Droid

**Comment by Dyras** [comment]
I get a crash when I enter any and all chats. Here's an excerpt from the debug log, if you want the whole thing just say the word

```
168419799; UID 2000; state: DISABLED
--------- beginning of crash
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: FATAL EXCEPTION: main
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: Process: com.afkanerd.deku, PID: 5703
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at com.example.swob_deku.Models.SIMHandler.getSubscriptionName(SIMHandler.java:63)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at com.example.swob_deku.Models.Messages.SingleMessagesThreadRecyclerAdapter.onBindViewHolder(SingleMessagesThreadRecyclerAdapter.java:259)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7254)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1631)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:643)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onMeasure(RecyclerView.java:3686)
11-05 18:53:32.726 10421  5703  5703 E AndroidRuntime: 	at android.view.View.measure(View.java:26368)
```

**Comment by sherlockwisdom** [comment]
Fixed with latest version 0.26.0
